xml/obs_authorization_token.xml

```diff
@@ -68,23 +68,31 @@
     service for that can be setup with
    </para>
 <screen><command>osc add git://....</command></screen>
+  <para>
+  The best way to create a token is bind it to a specific package. The advantage is that
+  the operation is limited to that package, so less bad things can happen
+  when the token leaks.
+  </para>
+<screen><command>osc token --create &lt;PROJECT&gt; &lt;PACKAGE&gt;</command></screen>
+<para>
+  Also, you do not need to specify the package at execution time.
+  But keep in mind that such form only works when you run it on an as
+  checkout of a package. Both commands below do the same thing in a different way:
```

Comment:
It would be a bug if that command is not running without checkout. it works for me also without one ...